contracts/orderbook/IWalletFactor.sol

```diff
@@ -0,0 +1,30 @@
+pragma solidity ^0.8.4;
+interface IWalletFactory{
+    struct OpenOrder{
+        address trader;
+        bytes32 loanID;
+        address iToken;
+		address loanTokenAddress;
+        uint price;
+        uint leverage;
+        uint loanTokenAmount;
+        uint collateralTokenAmount;
+        bool isActive;
+        address base;
+        uint orderType;

```

Comment:
would be nice to have enum for order Type

Reply:
this has been implemented.